project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementService.java

```diff
@@ -695,7 +750,7 @@ protected List<Result<MeasurementId, ErrorCode>> performUpdate(
     }
     try {
       if (measurementMetadataList.get(0) instanceof ProteomicsMeasurementMetadata) {
-        return performUpdatePxp(measurementMetadataList, projectId);
+        return Collections.emptyList(); // TODO Fix for functionality
```

Comment:
What does this comment mean? Can you elaborate? I thought the update case is tackled by updateAll ?